### PR TITLE
Fix nxos tests failures

### PR DESCRIPTION
**Pull request by trishnaguha**
Signed-off-by: Trishna Guha <trishnaguha17@gmail.com>

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENTS
lib/ansible/plugins/action/nxos_file_copy.py
test/integration/targets/nxos_{linkagg,config,rollback}

### CODE REVIEW

File: lib/ansible/plugins/action/nxos_file_copy.py [hunk 1]
```diff
@@ -213,6 +213,7 @@ def file_push(self):
 
         remote_file = remote_file or os.path.basename(local_file)
         remote_exists = self.remote_file_exists(remote_file, file_system)
+        self.results['changed'] = False
```
Comment (mikewiebe) [inline lib/ansible/plugins/action/nxos_file_copy.py:216]: @trishnaguha What is the reason for this change?

**Review by mikewiebe — CHANGES_REQUESTED** [review]
@trishnaguha Had a question about a change you are making to nxos_file_copy.

**Comment by mattclay** [comment]
Closing and re-opening to attempt to fix issue with commits being found.

**Comment by mattclay** [comment]
GH appears to have issues with this PR. The easiest way to fix it is to close it and open a new PR.

File: lib/ansible/plugins/action/nxos_file_copy.py (hunk 1 above)
Comment (trishnaguha) [reply lib/ansible/plugins/action/nxos_file_copy.py:216]: @mikewiebe If there is no change made to the device the results dictionary should have the key `changed` set to False which is not the case for this module. 
We have integration tests failing for this https://github.com/ansible/ansible/blob/devel/test/integration/targets/nxos_file_copy/tests/cli/negative.yaml#L54
This is applicable both for file pull and file push.
